Cython/Compiler/Parsing.py

```diff
@@ -3521,13 +3522,15 @@ def p_c_class_definition(s, pos,  ctx):
         bases = bases,
         objstruct_name = objstruct_name,
         typeobj_name = typeobj_name,
+        check_size = check_size,
         in_pxd = ctx.level == 'module_pxd',
         doc = doc,
         body = body)
 
 def p_c_class_options(s):
     objstruct_name = None
     typeobj_name = None
+    check_size = b'min'
```

Comment:
I'd like to have this a normal text string rather than a bytes string. Comparison to bytes might fail then parsed from Py3 code with unicode literals.
One way to do that might be to use the identifier `min` instead of the string literal `"min"`. Or check the literal node type and decode at need.

Reply:
it is messy since we want min, True, or False. Changing from p_atom(s) to p_ident(s) makes True and False into strings not booleans, is that OK? In any case, that enables removing the `b` (as long as we use a bare `min` not `'min'`. Changed documentation (which was missing quotes anyway) and test too.